content/doc/book/managing/system-properties.adoc

```diff
@@ -1632,6 +1632,15 @@ properties:
   since: 2.196
   description: |
     `true` to disable GZIP compression of artifacts when they're transferred from agent nodes to controller.  Uses less CPU at the cost of increased network traffic.
+- name: jenkins.monitor.JavaVersionRecommendationAdminMonitor.disable
+  tags:
+  - monitor
```

Comment:
```suggestion
```

Doesn't really make sense to add a unique tag for one entry in this list. See the list on top of this document for a complete list of "supported" tags.